content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/develop-a-culture-of-support-and-staff-development.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Develop a culture of support and staff development
+layout: /markdown_resource.html.erb
+---
+
+# Develop a culture of support and staff development
+
+<strong class="govuk-tag">Case study</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** Kensington Primary School
+
+**Location:** Newham, East London
+
+**Phase:** Primary
+
+**Number of pupils:** 700
+
+**Contact details:** Email the school office at <info@kensington.ttlt.academy> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Investing time and support to help everyone develop pays off. This policy resulted in paperwork reducing greatly and a positive culture developing.
+      </p>
+       <p>
+         A year 5 teacher who had been teaching for six years stated:
+       </p>
+      <p>
+        “When I first arrived at the school the paperwork was intense. We had regular lesson observations, we were required to annotate daily planning, update assessment scoring systems, conduct book scrutinies, complete evidence trails and do lots of marking in books. Over the years the school has slowly moved away from, what at times seemed a box-ticking culture, to remove the burden of paperwork away from the teachers so they can focus on what makes a difference: working with the children. Now our approach to assessment and monitoring is much more informal and involves daily assessment in class and informal chats with our SLT and wider team. As a team we communicate daily and make changes based on what we see each day. This approach is having a real impact in class.” 
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ben Levinson, headteacher
+
+{inset-text}
+We believe that if we concentrate on supporting each other in a professional positive environment that we will achieve the best outcomes for children. Our results show this to be true, together with the fact that we have not had one exclusion in the school in the past seven years.  
+
+The focus is on support and development in a positive atmosphere, working together and taking care of each other.  
+
+Our underlying ethos guides everything we do: we strive to have an open and reflective approach that means everyone feels confident to give and receive feedback: collaborative and developmental, not judgmental and directive.  
```

Comment:
Rather than having two colons in the same sentence here, the first should be changed to a comma. The second should be a '-'